.github/workflows/test.yml

```diff
@@ -39,6 +39,7 @@ jobs:
 
       - name: Secrets check
         run: |
+          export TERM=dumb
```

Comment:
I really doubt this is actually helping work around the "say not installed" issue. But I also don't think there is any real harm in having it here. [Slack thread](https://broadinstitute.slack.com/archives/C03BCARP494/p1665766224753949) with context.